pkgs/tools/security/tracee/default.nix

```
@@ -0,0 +1,55 @@
+{ stdenvNoCC, lib, fetchzip, autoPatchelfHook, elfutils, zlib }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tracee";
+  version = "0.6.5";
+
+  src = fetchzip {
+    url = "https://github.com/aquasecurity/tracee/releases/download/v${version}/tracee.tar.gz";
+    sha256 = "sha256-VgDJp+NsAV32kwa/GTrLnVH0IaTclsT9aD77SmMxVNM=";
+    stripRoot = false;
+  };
+
+  strictDeps = true;
+  nativeBuildInputs = [ autoPatchelfHook ];
+  buildInputs = [ elfutils zlib ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,share/tracee}
+    mv dist/tracee-{ebpf,rules} $out/bin
+    mv dist/rules $out/share/tracee/
+
+    runHook postInstall
+  '';
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+
+    $out/bin/tracee-ebpf --help
+    $out/bin/tracee-ebpf --version | grep "v${version}"
+
+    $out/bin/tracee-rules --help
+
+    runHook postInstallCheck
+  '';
+
+  meta = with lib; {
+    homepage = "https://aquasecurity.github.io/tracee";
+    changelog = "https://github.com/aquasecurity/tracee/releases/tag/v${version}";
+    description = "Linux Runtime Security and Forensics using eBPF";
+    longDescription = ''
+      Tracee is a Runtime Security and forensics tool for Linux. It is using
+      Linux eBPF technology to trace your system and applications at runtime,
+      and analyze collected events to detect suspicious behavioral patterns. It
+      is delivered as a Docker image that monitors the OS and detects suspicious
+      behavior based on a pre-defined set of behavioral patterns.
+    '';
+    license = licenses.asl20;
+    maintainers = with maintainers; [ jk ];
+    platforms = [ "x86_64-linux" ];
```

The readme only says *Linux*. Nothing about x86_64.
```suggestion
    platforms = platforms.linux;
```

---

Since im fetching from a prebuilt release thats only built for x86_64 linux it has to be pinned

---

Another reason to use `buildGoModule` so we have support for multiple architectures ^^